core/src/main/resources/wrangler-finder.xml

```diff
@@ -1,201 +0,0 @@
-<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
if you are deleting this file, then why is the new one named with a -1 at the end? just name it the same thing?